src/glmfit.jl

```diff
@@ -158,19 +158,21 @@ mutable struct GeneralizedLinearModel{G<:GlmResp,L<:LinPred} <: AbstractGLM
     fit::Bool
 end
 
-function coeftable(mm::AbstractGLM)
+function coeftable(mm::AbstractGLM; level::Real=0.95)
     cc = coef(mm)
     se = stderror(mm)
     zz = cc ./ se
-    CoefTable(hcat(cc,se,zz,2.0 * ccdf.(Ref(Normal()), abs.(zz))),
-              ["Estimate","Std.Error","z value", "Pr(>|z|)"],
+    p = 2.0 * ccdf.(Ref(Normal()), abs.(zz))
```

Comment:
```suggestion
    p = 2 * ccdf.(Ref(Normal()), abs.(zz))
```
Avoids unnecessary promotion to `Float64`.